app/components/StyledCard/tests/index.test.js

```diff
@@ -8,26 +8,57 @@ import React from 'react';
 import { renderWithIntl } from '@utils/testUtils';
 import StyledCard from '../index';
 import { data } from './testData';
+import { fireEvent} from "@testing-library/react"
+import {BrowserRouter} from "react-router-dom"
 
 describe('<StyledCard />', () => {
-  
+
   const sampleData = {
     results: data
   }
 
   it('should render and match the snapshot', () => {
-    const { baseElement } = renderWithIntl(<StyledCard musicData={sampleData} />);
+    const { baseElement } = renderWithIntl(<BrowserRouter><StyledCard musicData={sampleData} /></BrowserRouter>);
```

Comment:
please move BrowserRouter to `renderWithIntl`, so we don't need to add it separately everywhere